PathingAPI/PPather/Triangles/MPQTriangleSupplier.cs

```diff
@@ -171,6 +171,10 @@ public static string[] GetArchiveNames(DataConfig dataConfig, Action<string> log
 
         public void SetContinent(string continent)
         {
+            if (continent == null) {
+                return;
+            }
+
```

Comment:
this one sounds like a workaround rather then an actual fix. By looking at the `Json/WorldToMap/WorldMapArea.json`

Reply:
yes, but its too difficult to find info about locations, your json is not so good, alterac for example has 2 maps.

Reply:
All the data is coming from the following tables.

* data/uimap.csv
https://wow.tools/dbc/?dbc=uimap&build=2.5.1.38364&hotfixes=true#page=1

* data/worldmaparea.csv
https://wow.tools/dbc/?dbc=worldmaparea&build=2.4.3.8606#page=1